### ISSUE TITLE
Can no longer deploy pods when dynatrace operator is monitoring namespace

### ISSUE THREAD

**Issue by patrickpichler** [Dynatrace/dynatrace-operator#995]
When trying to deploy a pod to a namespace which is monitored by the dynatrace operator, 
the init containers crashes with
```
"level":"info","ts":"2022-07-21T15:36:48.950Z","logger":"standalone-init","msg":"checking envvars"}
{"level":"info","ts":"2022-07-21T15:36:48.950Z","logger":"standalone-init","msg":"DT_WORKLOAD_KIND environment variable is missing"}
{"error":"1 envvars missing","level":"error","logger":"main","msg":"initContainer command failed","stacktrace":"1 envvars missing
github.com/Dynatrace/dynatrace-operator/src/standalone.(*environment).setRequiredFields
        /app/src/standalone/env.go:74
github.com/Dynatrace/dynatrace-operator/src/standalone.newEnv
        /app/src/standalone/env.go:45
github.com/Dynatrace/dynatrace-operator/src/standalone.NewRunner
        /app/src/standalone/run.go:30
main.startStandAloneInit
        /app/src/cmd/operator/standalone.go:11
main.main
        /app/src/cmd/operator/main.go:85
runtime.main
        /usr/local/go/src/runtime/proc.go:250
runtime.goexit
        /usr/local/go/src/runtime/asm_amd64.s:1571","ts":"2022-07-21T15:36:48.950Z"}⏎  
```

The bug occurs in operator version`0.7.2`.
In version `0.4.1` pod deployment is working. 

We are using `cloudNativeFullStack` monitoring.

**Comment by chrismuellner** [comment]
Did you create this pod manually without an owner reference, i.e. just pod definition?

In that case it's a known issue that will be fixed in v0.8.0. As a workaround you can create the pod via a deployment.

**Comment by yeahservice** [comment]
It was just a pod definition without a deployment. But good to know that a fix is coming soon.
